src/sections/Books-grid/index.js

```diff
@@ -41,28 +40,60 @@ const BooksGrid = ({hide_path, limit}) => {
     );
 
     let path = hide_path ? "" : "Books";
-    let no_of_items = limit ? limit : data.length;
+    let thumb_imgs = [];
+    {data.allMdx.nodes.map(({frontmatter}) => (
+        thumb_imgs.push(frontmatter.thumbnail.publicURL)
+    ));}
+
+    var settings = {
```

Comment:
`const` should be used

Reply:
It is done purposefully, because the same variable is used in the 'responsive' array within that variable.